### ISSUE TITLE
Pd embedded on OSX

### ISSUE THREAD

**Issue by kmatheussen** [kmatheussen/radium#69]

**Comment by esaruoho** [comment]
Any chance this will be implemented eventually into Radium 3?

**Comment by kmatheussen** [comment]
Yes. Pd should be included in both windows and osx as soon as I convert Radium to use the new multi-dsp system in Pd that Miller made last year.
